docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -418,7 +418,14 @@ the property may be configured for:
      - Any S3-compatible storage
    * - ``exchange.gcs.json-key-file-path``
      - Path to the JSON file that contains your Google Cloud Platform
-       service account key.
+       service account key. Not to be set together with
+       ``exchange.gcs.json-key``
+     -
+     - GCS
+   * - ``exchange.gcs.json-key``
+     - Content of the JSON file that contains your Google Cloud Platform
```

Comment:
"Content of the JSON file" implies to me that you both set this property **and** have a JSON file installed on the cluster, but that doesn't sound like the intention.

Would suggest something like "Your Google Cloud Platform service account key in JSON format" or similar, that doesn't make reference to a file.